src/Tools/dotnet-monitor/Startup.cs

```diff
@@ -209,5 +213,25 @@ private static void LogBoundAddresses(IFeatureCollection features, AddressListen
                 logger.BoundMetricsAddress(metricAddress);
             }
         }
+
+        private static void LogElevatedPermissions(IAuthOptions options, ILogger logger)
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                WindowsIdentity currentUser = WindowsIdentity.GetCurrent();
+                WindowsPrincipal principal = new WindowsPrincipal(currentUser);
+                if (principal.IsInRole(WindowsBuiltInRole.Administrator))
+                {
+                    logger.ExcessivePermissions();
+                    // In the future this will need to be modified when ephemeral keys are setup
```

Comment:
I think your modification covers the future temporary key scenario as well; in temporary key mode, we should still probably turn on ntlm on windows, but we will disable it in admin mode.

Reply:
That's what I thought, if this doesn't work we can modify `options.EnableNegotiate` if necessary.